src/app/window-handler.ts

```diff
@@ -785,6 +792,7 @@ export class WindowHandler {
         }
 
         this.screenSharingIndicatorWindow = createComponentWindow('screen-sharing-indicator', opts);
+        this.moveWindow(this.screenSharingIndicatorWindow, topPositionOfIndicatorScreen);
```

Comment:
@johankwarnmarksymphony `topPositionOfIndicatorScreen` where is this value coming from?

Reply:
Got it, It's a constant with a value of `16`

Reply:
Yes fixed value from the designer Oliver.